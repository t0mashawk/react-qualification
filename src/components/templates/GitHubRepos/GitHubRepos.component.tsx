import React from 'react'
import * as Styled from './GitHubRepos.styles';

interface PageProps {
  children: React.ReactNode;
}

function GitHubRepos(props: PageProps): JSX.Element {
  const { children } = props;

  return <Styled.Container>{children}</Styled.Container>;
}

export default GitHubRepos
