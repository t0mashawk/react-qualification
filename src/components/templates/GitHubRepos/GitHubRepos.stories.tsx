import React from "react";
import { storiesOf } from "@storybook/react";
import GitHubRepos from "./GitHubRepos.component"

storiesOf("GitHubRepos", module)
    .add("test",
        () => <GitHubRepos>
            <p>Here are github repos</p>
        </GitHubRepos>
    )