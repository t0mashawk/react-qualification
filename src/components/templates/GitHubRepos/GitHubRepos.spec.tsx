import React from "react";
import renderer from "react-test-renderer";

import GitHubRepos from "./GitHubRepos.component"

test("GitHubRepos renders as expected", () => {

  const component = renderer.create(<GitHubRepos> <p>Here are github repos</p></GitHubRepos>);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});