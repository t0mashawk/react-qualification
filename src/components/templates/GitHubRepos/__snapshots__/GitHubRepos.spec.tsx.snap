// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubRepos renders as expected 1`] = `
<div
  className="sc-AxjAm jFTkVl"
>
   
  <p>
    Here are github repos
  </p>
</div>
`;
